.github/workflows/build.yml

```diff
@@ -32,6 +32,6 @@ jobs:
       - name: Uploads the generated pdf
         uses: actions/upload-artifact@v2
         with:
-          name: spearbit.pdf
+          name: spearbit-${{ github.event.repository.name }}-${{ github.sha }}.pdf
```

Comment:
BTW this should be a ZIP. Should we change it to zip?

Reply:
It auto-inserts the `.zip` extension, so if I added it here it would be `spearbit.zip.zip`. I think `.pdf.zip` should be fine